src/varnish.c

```diff
@@ -852,19 +852,19 @@ static int varnish_monitor(void *priv,
       return varnish_submit_derive(conf->instance, "mempool",
                                    "total_operations", "frees", val);
     else if (strcmp(name, "recycle") == 0)
-      return varnish_submit_derive(conf->instance, "mempool",
```

Comment:
I've also changed to "submit as gauge type" at the `mempool` section where the counters have type `objects`